tests/e2e/bin/local-env/env-check.sh

```diff
@@ -9,5 +9,5 @@ set -u
 # If the WordPress container isn't running, start the environment.
 if ! container ls > /dev/null ; then
 	status_message "E2E environment is not running, starting it now..."
-	npm run env:start
+	npm start
```

Comment:
This recently changed in another PR and wasn't updated. This is now called by `pretest` in the e2e workspace so it needed to be updated to call the new (unprefixed) script.